### PR TITLE
Refresh Codemagic guide

### PR DESCRIPTION
**Pull request by bryanoltman**
## Status

**READY**

## Description

Updates the Codemagic guide to include walkthroughs for releasing, patching, and publishing both iOS and Android.

### CODE REVIEW

File: src/content/docs/ci/_authentication.mdx [hunk 1]
```diff
@@ -20,13 +25,14 @@ Waiting for your authorization...
 
 🎉 Success! Use the following token to login on a CI server:
 
-<SHOREBIRD_TOKEN>
+qwerasdf1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqwer1234asdfqw==
```
Comment (felangel) [inline src/content/docs/ci/_authentication.mdx:28]: 😆

File: src/content/docs/ci/_authentication.mdx [hunk 2]
```diff
@@ -20,13 +25,14 @@ Waiting for your authorization...
...
 
 Example:
 
 export SHOREBIRD_TOKEN="$SHOREBIRD_TOKEN" && shorebird patch android
 ```
 
 :::caution
-The `SHOREBIRD_TOKEN` is a secret and should not be committed directly in your source code or shared publicly.
+The `SHOREBIRD_TOKEN` is a secret and should not be check in to your source
```
Comment (felangel) [inline src/content/docs/ci/_authentication.mdx:36]:
```suggestion
The `SHOREBIRD_TOKEN` is a secret and should not be checked in to your source
```

File: src/content/docs/ci/codemagic.mdx [hunk 3]
```diff
@@ -6,312 +6,500 @@ sidebar:
...
+## Prerequisites
 
-✅ You are logged into a Shorebird account.
+This guide assumes that:
 
-:::note
-Refer to the [getting started](/) instructions for more information.
-:::
+1. Shorebird CLI is installed on your machine ([guide](https://docs.shorebird.dev/#install)).
+2. Codemagic is connected to your repository ([guide](https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/#adding-the-app-to-codemagic)).
```
Comment (felangel) [inline src/content/docs/ci/codemagic.mdx:37]:
```suggestion
1. Codemagic is connected to your repository ([guide](https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/#adding-the-app-to-codemagic)).
```

File: src/content/docs/ci/_authentication.mdx (hunk 1 above)
Comment (bryanoltman) [reply src/content/docs/ci/_authentication.mdx:28]: Trying to make it look realistic! 😛

**Review by felangel — APPROVED** [review]

**Pull request merged**